packages/css/src/components/description-list/description-list.scss

```
@@ -54,8 +62,22 @@
   }
 }
 
-@mixin reset-dd {
-  margin-inline: 0;
+.ams-description-list__section {
+  @media screen and (min-width: $ams-breakpoint-medium) {
+    column-gap: var(--ams-description-list-column-gap);
+    display: grid;
+    grid-column: span 2;
+
+    :only-of-type {
```

This would apply to all descendants of the section; we intend to target the terms and descriptions.

```suggestion
    > :only-of-type {
```